test/invoice_repository_test.rb

```diff
@@ -101,9 +101,42 @@ def test_finds_all_by_updated_at
     assert_equal 0, result2.length
   end
 
+  def test_create_creates_a_new_invoice
```

Comment:
Assignment Branch Condition size for test_create_creates_a_new_invoice is too high. [22.02/15]